svthalia/Reaxit
# Display event organizers

Closes #346.

### Summary
Changed `Event` and related types to include a `List<ListGroup>` of organisers.
Changed database query parsing to also parse organisers.
Changed `event_screen` to display organisers in a field similar to location.

### How to test
Steps to test the changes you made:
1. Go to the home screen.
2. Click on any event.
3. See the new field.
